src/bin/cql-stress-cassandra-stress/java_generate/distribution/fixed.rs

```diff
@@ -23,3 +27,36 @@ impl Distribution for FixedDistribution {
 
     fn set_seed(&self, _seed: i64) {}
 }
+
+pub struct FixedDistributionFactory(pub i64);
+
+impl DistributionFactory for FixedDistributionFactory {
+    fn create(&self) -> Box<dyn Distribution> {
+        Box::new(FixedDistribution::new(self.0))
+    }
+}
+
+impl FixedDistributionFactory {
+    pub fn parse_from_description(desc: Description<'_>) -> Result<Box<dyn DistributionFactory>> {
+        let result = || -> Result<Box<dyn DistributionFactory>> {
```

Comment:
Nit: if `result` were the name of the _result_ of the lambda, then I'd be fine with it, but it's a name for the _lambda_ itself. That's slightly confusing.